### DISCUSSION TITLE
V-DH doesn't work anymore on MaxStream.video

### DISCUSSION THREAD

**Discussion by a-ghigi** [aclap-dev/video-downloadhelper#1458] [Help and Q&A]
### Your question

Since a few weeks, Video Downloadhelper doesn't work anymore on MaxStream.video

It keeps detecting 20 seconds chunks, one after another, whereas before it detected the full stream

Here's a link to test with: https://maxstream.video/emvvv/7w8b0zxqtjuk

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: it
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"6bdc7f11592d45c0"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "/Users/angeloghigi/dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/angeloghigi/dwhelper"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 5,
    "successfull_dl": 81,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_5181642069833712"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "The_Holdovers_-_Lezioni_di_vita_HD_2023_Bluray_720p.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/The_Holdovers_-_Lezioni_di_vita_HD_2023_Bluray_720p.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://maxlinks.online/watchfree/79xxeh3hm3xz/m62qy42ff5oy/UmpZa3pNdEFWUmJTRytGNDRqQ3k0Zz09"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1728972783512
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8520208986954659"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "007_-_Casino_Royale_HD_2006_Bluray.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/007_-_Casino_Royale_HD_2006_Bluray.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://maxlinks.online/watchfree/4twsfwas696s/z0gd6y0o4753/UmpZa3pNdEFWUmJTRytGNDRqQ3k0Zz09"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729182453492
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_5612081307922597"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": " El sabor de las cosas simples.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/ El sabor de las cosas simples.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.filmin.es/player?type=film&mediaId=47226"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729507242348
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_6734912685826431"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Hellboy.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/Hellboy.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://maxlinks.online/watchfree/wi29crlo9wrk/pj0izylxa3gr/UmpZa3pNdEFWUmJTRytGNDRqQ3k0Zz09"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729981706953
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8879298666375084"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "MixDrop - SUB ITA Hellboy The Crooked Man 2024 WEBRip AC3 CB01.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/MixDrop - SUB ITA Hellboy The Crooked Man 2024 WEBRip AC3 CB01.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://mixdrop.club/f/l7j9x7nquqr18np/2/_SUB_ITA_Hellboy_The_Crooked_Man_2024_WEBRip_AC3_CB01.mp4"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729983064423
                        }
                    }
                ]
            }
        ]
    },
    "record_download_history": true
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

My environment is Windows 11 Pro 64-bit version 24H2 build 26100.2314, Firefox 132.0.2 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

I visited the web page whose URL you supplied.  VDH recognized nothing.  So I let the video play in the web page for about 2 seconds then paused playback.  That made VDH recognize something.  I downloaded it without incident.  The download speed gradually ramped up to a little under 12 million bytes per second (observed in the Windows Resource Monitor), quite an acceptable speed.  The file I got was about 946M & it downloaded in about a minute & a half.  It played fine in VLC, although I skimmed it, only sampling it at intervals.  It had video & audio right to the end of the full 1:41:16 duration.

There are several differences between your environment & mine.  You're on Mac, I'm on Windows.  You're using Chrome, I'm using Firefox.  You're using a general availability VDH, I'm using a beta (which is available only on Firefox).  My best guess is that the problem lies in the browser.  This would not be the first time something worked better in Firefox than in Chrome.  I'm sure you're not willing to switch browsers.  Still, it would be an interesting experiment to see if the VDH beta worked on your system.  All I can say is it worked on mine.

**Comment by a-ghigi** [comment]
> My best guess is that the problem lies in the browser

Tried with Firefox, got the same result

> I'm using a beta

Maybe the next stable release will fix the problem
Meanwhile I found a workaround using JDownloader 2

**Addon & CoApp version and details**

version: 9.1.0.44
target: mozilla
channel: stable
lang: it
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"mismatch":true,"key":"6bdc7f11592d45c0","other_browser":"Chrome","this_browser":"Firefox"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:132.0) Gecko/20100101 Firefox/132.0
{
    "download_directory": "/Users/angeloghigi/dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 1,
    "successfull_dl": 2
}

**Comment by a-ghigi** [comment]
> Maybe the next stable release will fix the problem

Uhm, now I'm using Version 9.1.1.0 (stable) and still has the same problem


**Addon & CoApp version and details**

version: 9.1.1.0
target: google
channel: stable
lang: it
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"6bdc7f11592d45c0"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "/Users/angeloghigi/dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/angeloghigi/dwhelper"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 8,
    "successfull_dl": 85,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_7383734372234749"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "QU3 H3 H3CH0 Y0 P4R4 M3R3C3R 3ST0.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/QU3 H3 H3CH0 Y0 P4R4 M3R3C3R 3ST0.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://ok.ru/video/885241678479"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1731437336692
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3695467407876288"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Dailymotion.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/Dailymotion.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.dailymotion.com/video/x8ow8k"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1733908669898
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8593798220577510"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Видео La concejala antropofaga Exploradoresp2p OK RU.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/Видео La concejala antropofaga Exploradoresp2p OK RU.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://m.ok.ru/video/4787134335576"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1733908890205
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_321598173281809"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Dailymotion-01.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "/Users/angeloghigi/dwhelper/Dailymotion-01.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.dailymotion.com/video/x8ow8k"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1733910563647
                        }
                    }
                ]
            }
        ]
    },
    "record_download_history": true
}
